frontend/src/pages/projects/screens/spawner/storage/StorageField.tsx

```diff
@@ -56,7 +57,7 @@ const StorageField: React.FC<StorageFieldType> = ({ storageData, setStorageData,
                   setData={(data) => setStorageData('existing', data)}
                   editStorage={editStorage}
                   selectDirection="up"
-                  menuAppendTo={getDashboardMainContainer()}
+                  // menuAppendTo={getDashboardMainContainer()}
```

Comment:
Was this necessary to remove? If so, remove commented code.

I'm curious as to why it's removed here but added in for `CreateNewStorageSection`.